### PR TITLE
Add cron jobs section

### PR DESCRIPTION
**Pull request by MadamaMktg**
Added cron jobs section :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +40,16 @@ List of links to the various checkers out there on the web for sites, domains, s
 | https://saucelabs.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 | https://testingbot.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 
+## Cron Jobs 🌎
+
+| URL | Description | 💳 |
+| --- | ----------- | -- |
+| https://healthchecks.io/ | Simple and Effective Cron Job Monitoring. | Free |
```
Comment (Brunty) [inline README.md:47]: Can these be put into alphabetical order? Both sections and links within them should be. Thanks!

https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md#getting-started

File: README.md [hunk 2]
```diff
@@ -39,6 +40,16 @@ List of links to the various checkers out there on the web for sites, domains, s
 | https://saucelabs.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 | https://testingbot.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 
+## Cron Jobs 🌎
```
Comment (Brunty) [inline README.md:43]: Can the icon be updated to match the one in the list above? ⚡

**Review by Brunty — CHANGES_REQUESTED** [review]

**Comment by MadamaMktg** [comment]
Updated the domain order to be alphabetical and also updated the icon. Thank you! 👍

**Review by Brunty — APPROVED** [review]

**Comment by Brunty** [comment]
Thank you for this! :)

**Pull request merged**